test/e2e/default-fixture.js

```diff
@@ -261,6 +261,11 @@ function defaultFixture(inputChainId = CHAIN_IDS.LOCALHOST) {
           },
         },
       },
+      SnapController: {
```

Comment:
Do we need this? I'm not sure what the implications of adding this are since it seemingly modifies Sentry snapshots?